dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/event/ProgramRuleController.java

```diff
@@ -86,14 +82,18 @@ public void validateCondition( @RequestBody String condition, @RequestParam Stri
         }
         else
         {
-            message.setDescription(
-                ObjectUtils.firstNonNull( result.getErrorMessage(), result.getException().getMessage() ) );
```

Comment:
Usage of `firstNonNull` would actually result in a NPE since `result.getException()` would be `null`.